packages/devtools_app/lib/src/debugger/hover.dart

```diff
@@ -91,9 +92,9 @@ class HoverCardData {
   HoverCardData({
     this.title,
     @required this.contents,
-    this.width = HoverCardTooltip.defaultHoverWidth,
+    double width,
     this.position = HoverCardPosition.cursor,
-  });
+  }) : width = width ?? HoverCardTooltip.defaultHoverWidth;
```

Comment:
this change is surprising - why is it neccesary?

Reply:
because the default value is now not a constant and you can only use constants as default values for Dart constants.